lib/common/tests/strings/pcmk__str_in_list_test.c

```diff
@@ -40,13 +40,14 @@ star_matches(void) {
     GList *list = NULL;
 
     list = g_list_prepend(list, (gpointer) "*");
+    list = g_list_append(list, (gpointer) "more");
 
-    g_assert_true(pcmk__str_in_list(list, "xxx", pcmk__str_none));
-    g_assert_true(pcmk__str_in_list(list, "yyy", pcmk__str_none));
-    g_assert_true(pcmk__str_in_list(list, "XXX", pcmk__str_casei));
-    g_assert_true(pcmk__str_in_list(list, "", pcmk__str_none));
-    g_assert_true(pcmk__str_in_list(list, NULL, pcmk__str_none));
-    g_assert_true(pcmk__str_in_list(list, NULL, pcmk__str_null_matches));
+    g_assert_true(pcmk__str_in_list(list, "xxx", pcmk__str_star_matches));
+    g_assert_true(pcmk__str_in_list(list, "yyy", pcmk__str_star_matches));
+    g_assert_true(pcmk__str_in_list(list, "XXX", pcmk__str_star_matches|pcmk__str_casei));
+    g_assert_true(pcmk__str_in_list(list, "", pcmk__str_star_matches));
+    g_assert_true(pcmk__str_in_list(list, NULL, pcmk__str_star_matches));
+    g_assert_true(pcmk__str_in_list(list, NULL, pcmk__str_star_matches|pcmk__str_null_matches));
```

Comment:
We have some cases without star_matches as well

Reply:
I meant "should have"

Reply:
The star_doesnt_match function should take care of those cases.  Is there something else I'm missing?

Reply:
Ah I missed that, thanks